CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+### Added
+- Provide DOT graph of dependencies in the container.
```

Comment:
fx.DotGraph is now provided to the container by default. It contains a
    visualization of the dependency graph in a running Fx application.

And let's add something similar to this in the docs for DotGraph itself
because we haven't yet documented anywhere that this is happening.